src/Paket.Core/FrameworkHandling.fs

```diff
@@ -491,22 +609,27 @@ module KnownTargetProfiles =
        DotNetFrameworkIdentifiers
        |> List.map SinglePlatform
 
-    let DotNetStandardVersions =
-       [DotNetStandardVersion.V1_0
+    let DotNetStandardVersions = [
+        DotNetStandardVersion.V1_0
         DotNetStandardVersion.V1_1
         DotNetStandardVersion.V1_2
         DotNetStandardVersion.V1_3
         DotNetStandardVersion.V1_4
         DotNetStandardVersion.V1_5
-        DotNetStandardVersion.V1_6]
+        DotNetStandardVersion.V1_6
+        DotNetStandardVersion.V2_0
+    ]
         
 
     let DotNetStandardProfiles =
        DotNetStandardVersions
        |> List.map (DotNetStandard >> SinglePlatform)
        
-    let DotNetCoreVersions =
-       [DotNetCoreVersion.V1_0 ]
+    let DotNetCoreVersions = [
+        DotNetCoreVersion.V1_0
+        DotNetCoreVersion.V1_1
+        DotNetCoreVersion.V2_0
```

Comment:
I started out on this trying to fix https://github.com/fsprojects/Paket/issues/2245, but I haven't managed to figure it out yet. Any ideas?